src/tox/config/__init__.py

```diff
@@ -1053,6 +1053,11 @@ def get_envbindir(self):
             isinstance(self.python_info, NoInterpreterInfo)
             or tox.INFO.IS_WIN is False
             or self.python_info.implementation == "Jython"
+            or (
+                # this combination is MSYS2
+                tox.INFO.IS_WIN
+                and self.python_info.ossep == "/"
```

Comment:
Please rename it to os_sep to be snake case

Reply:
I was mimicking the existing `sysplatform`. I can change it though, if you like.

Reply:
Yes, please.